lib/web/scripts/install.go

```diff
@@ -0,0 +1,158 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package scripts
```

Comment:
Are we going to re-use these for other scripts? Eg, auto discover
If so, than we should probably move most code to a shared location.

Reply:
This code is building the right install options from the proxy point of view. The other services won't have access to this info and will have to do things a bit differently (query the proxy /find endpoint). I'll do a shared package in `lib/autoupdate/installscript` when I'll write the discovery service logic 👍